examples/plugins/.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {
+        "@nx/enforce-module-boundaries": ["off"]
```

Comment:
Why turn this off?

Reply:
because in the custom plugins I rely on direct imports ATM
`import { formatBytes, pluralize } from '../../../dist/packages/utils';`

When we have published the package we can change them